validator/src/main/resources/expected-data-template/otelSDKexpectedHTTPTrace.mustache

```diff
@@ -18,7 +18,7 @@
       "name": "HTTP GET",
       "http": {
         "request": {
-          "url": "https://aws.amazon.com/",
+          "url": "https://aws\\.amazon\\.com/",
```

Comment:
you might need to change all the trace templates?

Reply:
Ah I missed all the ones nested in packages. Good catch!

Reply:
Also FWIW, even if a period wasn't escaped like this, it wouldn't break validation. Because the unescaped period matches any character, including a literal period, so the correct endpoint would still be considered a match. But of course it's still better to escape them and match it exactly.

Reply:
gotcha!